php/class-post-type.php

```diff
@@ -341,11 +353,15 @@ public function render_data_metabox( $post ) {
 
 			$frontend_view_url = get_permalink( $post->ID );
 			echo sprintf(
-				'<a href="%s" class="button button-secondary">%s</a>',
+				'<a href="%s" class="button button-secondary">%s</a> ',
```

Comment:
If it's just to add visual space to the buttons then it should probably be done with CSS on `#snapshot-fork` instead.